shell/models/provisioning.cattle.io.cluster.js

```diff
@@ -1023,4 +1052,183 @@ export default class ProvCluster extends SteveModel {
   get fullDetailPageOverride() {
     return true;
   }
+
+  async loadAutoscalerEvents() {
+    const autoscalerConfigMap = await this.loadAutoscalerConfigMap();
+    const eventSchema = this.$rootGetters['management/schemaFor'](EVENT);
+    const fields = [new PaginationFilterField({
+      field: 'involvedObject.uid',
+      value: autoscalerConfigMap.metadata.uid,
+      exact: true,
+    }),
+    new PaginationFilterField({
+      field: 'metadata.namespace',
+      value: 'kube-system',
+      exact: true,
+    })];
+    const pagination = {
+      page:     1,
+      pageSize: 200,
+      filters:  [
+        new PaginationParamFilter({ fields })
+      ]
+
+    };
+    const params = stevePaginationUtils.createParamsForPagination({ schema: eventSchema, opt: { pagination } });
+    const url = `/k8s/clusters/${ this.mgmtClusterId }/v1/${ EVENT }?${ params }`;
+
+    const events = (await this.$dispatch('cluster/request', { url }, { root: true }))?.data || [];
```

Comment:
See `loadAutoscalerConfigMap` comments for details on why we're using `cluster/request`.

I will also note that I tested this with `ui-sql-cache` both enabled and disabled. I thought it had to be enabled for the filter fields to work but that doesn't appear to be the case.